tgui/packages/tgui-panel/styles/goon/chat-dark.scss

```diff
@@ -1378,3 +1378,8 @@ em {
 .stat_br {
   color: #2ace53;
 }
+
+.clown {
+  color: #ff70c1;
+  font-family: 'Comic Sans MS', cursive, sans-serif;
+}
```

Comment:
There is already another entry named clown. This will recolour all clown speech

Reply:
nope, it's called clowntext and it's not used since it's 24px and bolded

Reply:
OH yeah there is the job name color that recolors your speech's name. yea thats conflicting